theme.json

```diff
@@ -48,7 +48,19 @@
 			"replyColumnGap": "2.349rem",
 			"commentGap" : "5rem",
 			"commentGapSmall" : "10px",
-			"commentGapMedium" : "1.25rem"
+			"commentGapMedium" : "1.25rem",
+			"button": {
+				":active": {
+					"border": {
+						"color": "#FFF7BC"
```

Comment:
I think we should move to the color pallet to enable the users to customize it.
I didn't find a way to customize it.

Reply:
> I didn't find a way to customize it.

Even if I added it to the color palette, I don't think WordPress offers any way to customize the border color of different button states. Did you find a place to do this?

Reply:
I think you're referring to the fill button style, correct? If so, there was an easy fix for that in https://github.com/Automattic/sensei-theme/pull/52/commits/cbc10bd4b6d8963f44a4f530c04737d76a26950e. Now when you change the background color, there is no more border. 🙌🏻

Reply:
> I think you're referring to the fill button style, correct? If so, there was an easy fix for that in [cbc10bd](https://github.com/Automattic/sensei-theme/commit/cbc10bd4b6d8963f44a4f530c04737d76a26950e). Now when you change the background color, there is no more border. 🙌🏻

Hmn, maybe not.

I am talking more about it:
[Screen Recording on 2022-10-27 at 14_32_34.webm](https://user-images.githubusercontent.com/38718/198359305-be9b4801-e4a9-4cd5-9bc9-7172e8df3de8.webm)

The border color is set on the theme.json Custom Field which doesn't enable the users to customize.
On the video case, I  update all green tones to red tones but the border is still green and I don't have a way to customize it. 

<img width="279" alt="Screen Shot 2022-10-27 at 14 36 10" src="https://user-images.githubusercontent.com/38718/198359814-c637e00e-e1ec-4aa1-874d-7496b9d8ffb5.png">

Reply:
OK, all set in https://github.com/Automattic/sensei-theme/pull/52/commits/62fc8be599540f625b7c19250c7bfbf76308ffff. (Sorry, I did a force push. 😐 )